packages/create-gasket-app/lib/index.d.ts

```diff
@@ -278,18 +281,19 @@ export interface ActionWrapperParams {
 }
 
 declare module '@gasket/core' {
-  import { CreateContext } from 'create-gasket-app';
 
   export interface HookExecTypes {
-    presetPrompt(context: CreateContext): Promise<void>;
+    presetPrompt(
+      context: CreateContext,
+      utils: {
+        prompt: PromptModule;
```

Comment:
Use the alias? Perhaps `CreatePrompt` per other comment
```suggestion
        prompt: CreatePrompt;
```